src/Corporation/Corporation.ts

```diff
@@ -196,9 +209,27 @@ export class Corporation {
     if (this.valuationsList.length > corpConstants.valuationLength) this.valuationsList.shift();
     let val = this.valuationsList.reduce((a, b) => a + b); //Calculate valuations sum
     val /= corpConstants.valuationLength; //Calculate the average
```

Comment:
this should be
```js
val /= this.valuationsList.length;
```
(only relevant within 10 cycles after creating a corp, but good to fix it now)

Reply:
Done